### PR TITLE
Make the docs a smidge clearer

### PR DESCRIPTION
**Pull request by pfif**

### CODE REVIEW

File: content/en/docs/what-is-grpc/introduction.md [hunk 1]
```diff
@@ -97,8 +97,8 @@ gRPC plugin in your chosen language, see the [protocol buffers documentation][pr
 While [protocol buffers][] have been available to open source users for some time,
 most examples from this site use protocol buffers version 3 (proto3), which has
 a slightly simplified syntax, some useful new features, and supports more
-languages. Proto3 is currently available in Java, C++, Dart, Python,
-Objective-C, C#, a lite-runtime (Android Java), Ruby, and JavaScript from the
+languages. Proto3 is currently available in Java (both normal and lite-runtime for Android Java), C++, Dart, Python,
+Objective-C, C#, Ruby, and JavaScript from the
```
Comment (easwars) [inline content/en/docs/what-is-grpc/introduction.md:101]: Nit: could you please ensure that these lines are still wrapped at 80-cols. Thanks.

**Comment by easwars** [comment]
Could you please sign the CLA @pfif

**Comment by easwars** [comment]
@ejona86 @temawi 
For folks not signing the CLA, do we have any policy on how long we keep these PRs open before closing them?

**Comment by temawi** [comment]
I'm not aware of any policy, but I'll close this one at least since it's unlikely that it will happen eight months later.

**Pull request closed without merging**